packages/react-core/src/components/Menu/examples/Menu.md

```diff
@@ -0,0 +1,757 @@
+cssPrefix: pf-c-menu
+propComponents: ['Menu', 'MenuItem', 'MenuList']
+ouia: true
+---
+
+import { BarsIcon, ClipboardIcon, CodeBranchIcon, LayerGroupIcon, CubeIcon, TableIcon, BellIcon } from '@patternfly/react-icons/dist/js/icons';
+
+## Examples
+
+### Basic
+
+```js
+import React from 'react';
+import { Menu, MenuItem, MenuList } from '@patternfly/react-core';
+
+class MenuBasicList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      activeItem: 0
+    };
+    this.onSelect = result => {
+      this.setState({
+        activeItem: result.itemId
+      });
+    };
+  }
+
+  render() {
+    const { activeItem } = this.state;
+    const menuItems = [
+      <MenuList>
+        <MenuListItem
+          component="button"
+          // id="default-link1"
+          to="#default-link1"
+          itemId={0}
+          isActive={activeItem === 0}
+        >
+          Action
+        </MenuListItem>
+        <MenuListItem to="#default-link2" itemId={1} isActive={activeItem === 1}>
+          Link
+        </MenuListItem>
+        <MenuListItem
+          isDisabled
+          component="button"
+          // id="default-link3"
+          to="#default-link3"
+          itemId={2}
+          isActive={activeItem === 2}
+        >
+          Disabled Action
+        </MenuListItem>
+        <MenuListItem isDisabled id="default-link4" to="#default-link4" itemId={3} isActive={activeItem === 3}>
+          Disabled Link
+        </MenuListItem>
+      </MenuList>
+    ];
+    return <Menu onSelect={this.onSelect} items={menuItems}></Menu>;
+  }
+}
+```
+
+### With icons
+
+```js
+import React from 'react';
+import { Menu, MenuItem, MenuList } from '@patternfly/react-core';
+import { CodeBranchIcon, LayerGroupIcon, CubeIcon } from '@patternfly/react-icons/dist/js/icons';
+
+class MenuIconsList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      activeItem: 0
+    };
+    this.onSelect = result => {
+      this.setState({
+        activeItem: result.itemId
+      });
+    };
+  }
+
+  render() {
+    const { activeItem } = this.state;
+    const menuItems = [
+      <MenuList>
+        <MenuListItem
+          component="button"
+          icon={<CodeBranchIcon />}
+          to="#default-link1"
+          itemId={0}
+          isActive={activeItem === 0}
+        >
+          From Git
+        </MenuListItem>
+        <MenuListItem
+          component="button"
+          icon={<LayerGroupIcon />}
+          to="#default-link2"
+          itemId={1}
+          isActive={activeItem === 1}
+        >
+          Container Image
+        </MenuListItem>
+        <MenuListItem component="button" icon={<CubeIcon />} to="#default-link2" itemId={2} isActive={activeItem === 2}>
+          Docker File
+        </MenuListItem>
+      </MenuList>
+    ];
+    return <Menu onSelect={this.onSelect} items={menuItems}></Menu>;
+  }
+}
+```
+
+### With expanded toggle
```

Comment:
opened an issue in core: https://github.com/patternfly/patternfly/issues/3599

should I remove this variant for now?